packages/plugin_platform_interface/test/plugin_platform_interface_test.dart

```diff
@@ -12,7 +12,7 @@ class SamplePluginPlatform extends PlatformInterface {
   static final Object _token = Object();
 
   static set instance(SamplePluginPlatform instance) {
-    PlatformInterface.verifyToken(instance, _token);
+    PlatformInterface.verify(instance, _token);
     // A real implementation would set a static instance field here.
   }
 }
```

Comment:
We should make a version of this that still uses verifyToken so we're still exercising the codepath that >100 plugins are likely currently using, to make sure we don't break it later.

Reply:
Done